rescript/get_data.py

```diff
@@ -12,13 +12,132 @@
 import shutil
 import gzip
 import warnings
+import requests
+import tarfile
 
 import qiime2
 from urllib.request import urlretrieve
 from urllib.error import HTTPError
 from q2_types.feature_data import RNAFASTAFormat
 
 
+def _unite_dois_to_urls(DOIs):
+    '''Generate UNITE urls, given their DOIs.'''
+    # Make DOIs iterable
+    DOIs = [DOIs] if isinstance(DOIs, str) else DOIs
```

Comment:
why not just do `list(DOIs)`?

Though I am not sure if this is needed either, depending on how you register the inputs to the action (e.g., a `List[Str]` should always give you a list even if there is just one, I think)

Reply:
but also this function would only handle one DOI request at a time, so maybe this is not even needed (i.e., the input would always be a single DOI str)

Reply:
This is a reliable way to make it work with with multiple DOIs. 

This function is now used internally in `_unite_get_url` where it will only look up one DOI at a time.

Simplify it?

Reply:
do you want to retrieve multiple DOIs though? the user would request a single version at a time.

Reply:
I now have this take a single DOI and output a single URL